lib/src/svg/parser_state.dart

```diff
@@ -580,28 +580,39 @@ class _Elements {
         fontSize: fontSize,
       )!,
     );
-    final Image image = await resolveImage(href);
-    final DrawableParent parent = parserState._parentDrawables.last.drawable!;
-    final DrawableStyle? parentStyle = parent.style;
-    final DrawableRasterImage drawable = DrawableRasterImage(
-      parserState.attribute('id', def: ''),
-      image,
-      offset,
-      parseStyle(
-        parserState._key,
-        parserState.attributes,
-        parserState._definitions,
-        parserState.rootBounds,
-        parentStyle,
-        currentColor: parent.color,
-        fontSize: parserState.fontSize,
-      ),
-      size: size,
-      transform: parseTransform(parserState.attribute('transform'))?.storage,
-    );
-    final bool isIri = parserState.checkForIri(drawable);
-    if (!parserState._inDefs || !isIri) {
-      parserState.currentGroup!.children!.add(drawable);
+
+    if(href.startsWith('http')&& href.endsWith('.svg')){
+      final DrawableRoot drawable = await resolveSvg(
+          href,
+          parserState.theme,
+          parserState._key,
+          parserState._warningsAsErrors);
+      parserState.currentGroup!.children!.addAll(drawable.children);
```

Comment:
I don't think this will work correctly for a bunch of possible values in viewBox. I'm concerned about adding support for this feature without having a bunch of cases that test and exercise different options here.

Reply:
@dnfield  logically it should work, if SVG parser create drawable for children successfully. can you suggest some use case for those I'll check.

Reply:
@dnfield can you please help me here with other cases.